andino_gazebo/urdf/andino.gazebo.xacro

```diff
@@ -0,0 +1,169 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" name="andino">
+
+  <xacro:include filename="$(find andino_description)/urdf/andino.urdf.xacro"/>
+
+  <xacro:macro name="friction" params="reference mu">
+    <gazebo reference="${reference}">
+      <mu1>${mu}</mu1>
+      <mu2>${mu}</mu2>
+      <kp>1000000.0</kp>
+      <kd>1.0</kd>
+      <minDepth>0.0001</minDepth>
+      <maxVel>1.0</maxVel>
+    </gazebo>
+  </xacro:macro>
+
+  <xacro:friction reference="front_right_wheel" mu="10"/>
+  <xacro:friction reference="front_left_wheel" mu="10"/> 
+  <xacro:friction reference="rear_caster_wheel_link" mu="0.000"/>
+
+   <xacro:if value="$(arg use_gazebo_ros_control)">
+    <ros2_control name="diff_controller" type="system">
+    <hardware>
+      <plugin>gazebo_ros2_control/GazeboSystem</plugin>
+    </hardware>
+    <joint name="left_wheel_joint">
+      <command_interface name="velocity">
+        <param name="min">-5</param>
+        <param name="max">5</param>
+      </command_interface>
+      <state_interface name="velocity"/>
+      <state_interface name="position"/>
+    </joint>
+    <joint name="right_wheel_joint">
+      <command_interface name="velocity">
+        <param name="min">-5</param>
+        <param name="max">5</param>
+      </command_interface>
+      <state_interface name="velocity"/>
+      <state_interface name="position"/>
+    </joint>
+  </ros2_control>
+    <gazebo>
+      <plugin filename="libgazebo_ros2_control.so" name="gazebo_ros2_control">
+          <parameters>$(find andino_control)/config/andino_controllers.yaml</parameters>
+          <!-- <robotNamespace>: The ROS namespace to be used for this instance of the plugin, defaults to robot name in URDF/SDF -->
```

Comment:
You can pass here also adition parameters for the controller manager, for example the use_sim_time``

```
<parameters>$(find andino_gazebo)/config/gazebo_ros2_control.yaml</parameters>
```

and within gazebo_ros2_control.yaml:

```yaml
controller_manager:
  ros__parameters:
    use_sim_time: true
```